heutagogy/views.py

```diff
@@ -55,7 +55,11 @@ def get(self):
         filters = [db.Bookmark.user == current_user.id]
         if url is not None:
             filters.append(db.Bookmark.url == urldefrag(url).url)
-        filters.append(db.Bookmark.tags.contains(tags))
+
+        # If Bookmark.url is null, filtering will yield no results
```

Comment:
Are comment and code below related?

Reply:
Typo: it should have been `Bookmark.tags is null, ...`